db/migration/1676376942081-links.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class links1676376942081 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        queryRunner.query(`
+        CREATE TABLE links (
+            id INT NOT NULL AUTO_INCREMENT,
+            sourceId VARCHAR(255),
+            target VARCHAR(2047) NOT NULL,
+            type ENUM("gdoc", "url") NOT NULL,
```

Comment:
How about renaming `type` => `linkType` and `context` => `componentType`? I feel like context is a bit vague and doesn't tell me what to expect here. If the context is a component it contains the tag name; maybe for an inline link it should be `span-link` to be fully consistent instead of `inline`?